src/client_info.py

```diff
@@ -1,10 +1,27 @@
 from dataclasses import dataclass
+import json
 
 from . import user_dto, bundle_info, subscription_info
+from . import manifest
 
 
 @dataclass
 class ClientInfo:
     basic: user_dto.UserDto
     subscription: subscription_info.SubscriptionInfo
     bundles: list[bundle_info.BundleInfo]
+
+    def to_json(self, prety_print: bool = True) -> str:
```

Comment:
Shouldn't the variable be named pretty_print instead of prety_print?

Reply:
True